recipes/quickstart/NotebookLlama/README.md

```diff
@@ -0,0 +1,82 @@
+## NotebookLlama: An Open Source version of NotebookLM
+
+[Listen to audio from the example here](./resources/_podcast.mp3)
+
+This is a guided series of tutorials/notebooks that can be taken as a reference or course to build a PDF to Podcast workflow. 
+
+You will also learn from my experimentation of using Text to Speech Models.
+
+It assumes zero knowledge of LLMs, prompting and audio models, everything is covered in their respective notebooks.
+
+### Outline:
+
+Here is step by step thought (pun intended) for the task:
+
+- Step 1: Pre-process PDF: Use `Llama-3.2-1B` to pre-process and save a PDF
+- Step 2: Transcript Writer: Use `Llama-3.1-70B` model to write a podcast transcript from the text
```

Comment:
did 8B fail to write a good transcript?

Reply:
It's subjective since I didn't do any quantitative reviews but 70B felt more creative when doing the first parse. 

Should I change to 8B?

Reply:
Added a note